vtm/src/org/oscim/layers/tile/buildings/BuildingLayer.java

```diff
@@ -146,6 +153,33 @@ public boolean process(MapTile tile, RenderBuckets buckets, MapElement element,
 
         ExtrusionStyle extrusion = (ExtrusionStyle) style.current();
 
+        boolean isErased = false;
+        if (element.isBuilding() || element.isBuildingPart()) {
```

Comment:
There is an identical check right below, how these work together?

Also how about users who don't care for that feature, how can they avoid such expensive checks in each `process` call?